### PR TITLE
Remove obsolete upload size section

### LINKED ISSUE #27672: Maximum upload size is ignored.
### Steps to reproduce

1. As an admin go to settings admin/storage.
2. In file handling, set maximum upload size to 2MB.
3. As a regular user go to files view and upload a file bigger than 2MB.

### Expected behaviour
The upload is forbidden.

### Actual behaviour
File is upload, nothing is checked.

### Server configuration

**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
7.0

**ownCloud version:** (see ownCloud admin page)
current master
{"installed":"true","maintenance":"false","needsDbUpgrade":"false","version":"10.0.0.6","versionstring":"10.0.0 beta2","edition":"Community","productname":"ownCloud"}

**Updated from an older ownCloud or fresh install:**
Fresh

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No.

**Are you using encryption:** 
No

**Logs**

### Client configuration

**Browser**
Chrome

**Logs**
```
```

Working fine in version 9.1.4.

### PR DESCRIPTION
**Pull request by PVince81**
## Description
Removes max upload size settings section.

The admin.js file had additional code related to "sharingaim", I checked back to stable8 and this element doesn't exist anywhere, so these might be relics from ancient deleted features.

## Related Issue
Fixes https://github.com/owncloud/core/issues/27672

## Motivation and Context
The web UI now uses Webdav PUT instead of HTTP POST, so there is no upload size limit any more.
We remove the settings section to avoid confusion.

## How Has This Been Tested?
- verify that the section has disappeared
- click through other settings sections and check that there are no warnings in owncloud.log about missing stuff

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

@tomneedham please review

### CODE REVIEW

File: apps/files/js/admin.js [hunk 1]
```diff
@@ -1,54 +0,0 @@
...
-	// find all radiobuttons of that group
-	var sharingaimGroup = $('input:radio[name=sharingaim]');
-	$.each(sharingaimGroup, function(index, sharingaimItem) {
-		// set all buttons to the correct state
-		sharingaimItem.disabled = !publicEnable;
-	});
-}
-
-function setUpload(val, success, failure) {
-	$.post(OC.generateUrl('apps/files/ajax/setupload.php'), {maxUploadSize: val})
```
Comment (PVince81) [inline apps/files/js/admin.js]: forgot to remove this ajax file, will do

**Comment by PVince81** [comment]
and maybe also remove the utility functions... will do

**Comment by PVince81** [comment]
Pushed, please recheck.

I also removed the "setUploadLimit" function that writes into htaccess

**Comment by PVince81** [comment]
Argh, can of works
```
Test\IntegrityCheck\CheckerTest.testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
Test\IntegrityCheck\CheckerTest.testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
```

If removing the section is really needed I could remove it but leave the utility function. With that, the integrity checker would still add the max upload section as before but it won't be modifiable in the UI.

Then we can find a way to cleanly and safely remove all this old code...

**Comment by PVince81** [comment]
will remove in 10.0.2

**Comment by tomneedham** [comment]
Seems the hashes are still broken? Or jenkins isn't happy. Otherwise 👍

**Comment by PVince81** [comment]
rebased... trying my luck 🍀

**Comment by PVince81** [comment]
Pfff what now...

```
13:57:05 1) Test\IntegrityCheck\CheckerTest::testWriteCoreSignatureWithValidModifiedHtaccessAndUserIni
13:57:05 Exception: /var/lib/jenkins/workspace/owncloud-core_core_PR-27929-QPW6O7RTY6AX6VS3JGV5ICSA2CJUGRPTRTT6HVWGLCZ3N4MMPKBA/tests/data/integritycheck/htaccessWithValidModifiedContent/core is not writable
13:57:05 
13:57:05 /var/lib/jenkins/workspace/owncloud-core_core_PR-27929-QPW6O7RTY6AX6VS3JGV5ICSA2CJUGRPTRTT6HVWGLCZ3N4MMPKBA/lib/private/IntegrityCheck/Checker.php:308
13:57:05 /var/lib/jenkins/workspace/owncloud-core_core_PR-27929-QPW6O7RTY6AX6VS3JGV5ICSA2CJUGRPTRTT6HVWGLCZ3N4MMPKBA/tests/lib/IntegrityCheck/CheckerTest.php:601
13:57:05 
13:57:05 --
13:57:05 
13:57:05 There was 1 failure:
13:57:05 
13:57:05 1) Test\IntegrityCheck\CheckerTest::testVerifyCoreSignatureWithValidModifiedHtaccessAndUserIniSignatureData
13:57:05 Failed asserting that Array &0 (
13:57:05     'EXCEPTION' => Array &1 (
13:57:05         'class' => 'OC\IntegrityCheck\Exceptions\InvalidSignatureException'
13:57:05         'message' => 'Signature could not get verified.'
13:57:05     )
13:57:05 ) is identical to Array &0 ().
13:57:05 
13:57:05 /var/lib/jenkins/workspace/owncloud-core_core_PR-27929-QPW6O7RTY6AX6VS3JGV5ICSA2CJUGRPTRTT6HVWGLCZ3N4MMPKBA/tests/lib/IntegrityCheck/CheckerTest.php:704
13:57:05 
```

not writable??

**Comment by PVince81** [comment]
moving to triage, this will likely take longer...

**Comment by DeepDiver1975** [comment]
> moving to triage, this will likely take longer...

fixed ....

**Review by DeepDiver1975 — APPROVED** [review]
maybe squash a bit ....

**Comment by PVince81** [comment]
wow, CI already ran for 7 hours and still going...

**Comment by DeepDiver1975** [comment]
job 7 was green - https://jenkins.owncloud.org/job/owncloud-core/job/core/job/PR-27929/7/

we can merge from my pov

**Comment by DeepDiver1975** [comment]
@PVince81 backport?

**Comment by PVince81** [comment]
stable10: https://github.com/owncloud/core/pull/28579

**Pull request merged**